CHANGELOG.md

```diff
@@ -33,13 +35,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Removed
 - Conda install instructions and version badge
 - Early fail for different Python versions in regular pipeline
+- Workaround for BoTorch hybrid recommender data type
 
 ### Deprecations
 - `Interval.is_finite` replaced with `Interval.is_bounded`
 - Specifying target configs without explicit type information is deprecated
-
-### Changed
-- New version of campaign user guide and basic example
+- Python 3.8 no longer supported
```

Comment:
I think this does not belong to "deprecations" but perhaps to "removed" (e.g. simply "Python 3.8 support"). The current meaning of he deprecations block is that it only contains stuff that **is going to be removed** but is currently still supported

Reply:
i agree, will change it

for some reason i was thinking we do not generally have `removed` sections, but they are actually in there